src/platform_impl/linux/x11/window.rs

```diff
@@ -386,7 +386,7 @@ impl UnownedWindow {
                 normal_hints.set_min_size(min_inner_size.map(Into::into));
                 normal_hints.set_max_size(max_inner_size.map(Into::into));
                 normal_hints.set_resize_increments(
-                    pl_attribs
+                    window_attrs
                         .resize_increments
                         .map(|size| size.to_physical::<u32>(scale_factor).into()),
```

Comment:
There's no need for map given that function handles that or the scaling is wrong when calling the `set_resize_increments` here? Could you add a comment about that at least? I do understand that the code was like that before, but it would be nice to ensure that and remove extra convertion, since it looks confusing.

Reply:
You might be confused by the two lines above which look like `normal_hints.set_max_size(max_inner_size.map(Into::into));` - but in fact I modelled `resize_incremets` exactly like min/max_inner_size.

In those lines above e.g. the `min_inner_size` is is defined as `let mut min_inner_size = window_attrs.min_inner_size.map(|size| size.to_physical::<u32>(scale_factor))`, it just needs to be a mut local to update it for the ability to make windows unresizable.

Reply:
I'm not saying that the logic is wrong or anything. I'm trying to say that `set_resize_increments` already does conversion to physical pixels with the `scale_factor` so the logic seems redundant and you can pass `resize_increments` directly?

There's a chance that I don't understand this code or scale factor isn't there.

Reply:
Here we call `normal_hints.set_resize_increments` which just accepts a pair of i32s, physical, not `window.set_resize_increments` which accepts the `Size` enum and does the conversion - so that on window creation there is only one `xconn.set_normal_hints` call to set all the initial hints at once.